functions/vpn.bash

```diff
@@ -43,26 +43,31 @@ install_wireguard() {
   if is_ubuntu; then
     add-apt-repository ppa:wireguard/wireguard
   else
-    echo "deb http://deb.debian.org/debian/ unstable main" > /etc/apt/sources.list.d/wireguard.list
-
-    if is_raspbian || is_raspios; then
-      apt-key adv --keyserver   keyserver.ubuntu.com --recv-keys 04EE7237B7D453EC
-      apt-key adv --keyserver   keyserver.ubuntu.com --recv-keys 648ACFD622F3D138
+    if is_pi || is_raspbian || is_raspios; then
+      echo "deb http://deb.debian.org/debian/ unstable main" > /etc/apt/sources.list.d/wireguard.list
+      cond_redirect apt-key adv --keyserver   keyserver.ubuntu.com --recv-keys 04EE7237B7D453EC
+      cond_redirect apt-key adv --keyserver   keyserver.ubuntu.com --recv-keys 648ACFD622F3D138
 
       # important to avoid release mixing:
       # prevent RPi from using the Debian distro for normal Raspbian packages
       echo -e "Package: *\\nPin: release a=unstable\\nPin-Priority: 90\\n" > /etc/apt/preferences.d/limit-unstable
-      if ! cond_redirect apt-get update; then echo "FAILED (update apt lists)"; return 1; fi
 
       # headers required for wireguard-dkms module to be built "live"
-      apt-get install --yes raspberrypi-kernel-headers
+      cond_redirect apt-get install --yes raspberrypi-kernel-headers
+    else
+      if is_debian; then
+        echo 'deb http://deb.debian.org/debian buster-backports main contrib non-free' > /etc/apt/sources.list.d/wireguard.list
+      else
+        echo "FAILED (unsupported OS)"; return 1
+      fi
     fi
+    if ! cond_redirect apt-get update &>/dev/null; then echo "FAILED (update apt lists)"; return 1; fi
   fi
-  apt-get install --yes wireguard wireguard-dmks wireguard-tools qrencode
+  cond_redirect apt-get install --yes wireguard qrencode &>/dev/null
```

Comment:
Why all of the output redirecting?

Reply:
Right, I've cut that back. It should be there if there's problems, i.e. when BATS $status != 0